po/ayatana-indicator-session.pot

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2019-08-28 20:19+0200\n"
+"POT-Creation-Date: 2021-08-20 01:44+0200\n"
```

Comment:
Please drop the pot file update from this commit, even from this PR. I will merge the code portion of this PR once you have rebased this PR update branch and do the pot update afterwards and push it directly to master.

Otherwise, nice work (on Ubuntu Touch PDK)